t/api/reverse_dependencies.t

```diff
@@ -0,0 +1,46 @@
+#!perl
+
+use strict;
+use warnings;
+use Test::More tests => 45;
+use Test::Fatal;
+use Ref::Util qw< is_hashref >;
+
+use lib '.';
+use t::lib::Functions;
+
+my $mc = mcpan();
+can_ok( $mc, 'reverse_dependencies' );
+can_ok( $mc, 'rev_deps' );
+
+# isa_ok( $dist, 'MetaCPAN::Client::Distribution' );
+
+my $module = 'MetaCPAN::Client';
+
+my $rs = $mc->reverse_dependencies($module);
+
+isa_ok $rs, 'MetaCPAN::Client::ResultSet';
```

Comment:
please use parenthesis (you use them for `can_ok` above and the mixture of styles is confusing)